setup.py

```diff
@@ -39,6 +41,12 @@
         ('share/jupyter/lab/extensions',
          'js/lab-dist',
          'jupyter-threejs-*.tgz'),
+        ('share/jupyter/labextensions/jupyter-threejs/',
```

Comment:
It's easier to use `**`, like https://github.com/jupyterlab/extension-cookiecutter-ts/blob/master/%7B%7Bcookiecutter.python_name%7D%7D/setup.py#L41-L42